### DISCUSSION TITLE
Have you considered making an XBox 360 port?

### DISCUSSION THREAD

**Discussion by Androxilogin** [TheElixZammuto/moonlight-xbox#98] [Ideas]
This would be soo awesome and surely bring a lot of energy back into the community.

**Comment by TheElixZammuto** [comment]
I'm not planning to make a 360 ports for many reasons, but three in particular on top of my head:

1. Porting the UWP is impossible: DirectX11, XAML, and almost everything used by Moonlight UWP is not available on X360
2. The only way to make it possible (by starting from scratch anyway) is to use unsupported toolchains made for homebrew apps with less documentation than the official one.
3. Xbox360 is one of the hardest console to mod to be homebrew-capable, requiring a non-trivial amount of experience with electronics and microsolding, which I do not have. 

In conclusion, the ratio between the time spent to ipotetically make it and the userbase I can reach is too little to make it worth making a whole other application just for modded 360
